src/main/java/com/code_intelligence/jazzer/junit/FuzzTestExecutor.java

```diff
@@ -42,16 +37,23 @@
 import org.junit.jupiter.api.extension.ExtensionContext;
 import org.junit.jupiter.api.extension.ReflectiveInvocationContext;
 
-abstract class FuzzTestExecutor {
+class FuzzTestExecutor {
```

Comment:
@fmeum As discussed, I removed the different `TestExecutors`, and `libFuzzerArgs` are now appended accordingly, respectively used as fake `argv0`. I'm somewhat unsure why the `execute` methods of the two implementations differed, though.

Reply:
The `--keep_going` flag is only evaluated after checking the `findingHandler`. As that  always returns `false`, it looks like execution should stop at the first finding. So you're right, the previous behavior differed between fuzz tests started by JUnit and our CLI. Was/is that intended?

Reply:
Ok, so it was intended. As there is probably no idiomatic way to support something like `keep_going` in JUnit (why would there) the different behavior seems reasonable. 

I only passed down a flag to distinguish the two modes, as the two implementations were almost identical. That being said, it's not super clean. Having multiple implementations or other means to configure `execute` seemed similar overkill.